addon/components/polaris-callout-card.hbs

```diff
@@ -55,12 +57,13 @@
                   />
                 </PolarisButtonGroup>
               {{else}}
-                <PrimaryButton />
+                <PrimaryButton data-test-callout-card-primary-button />
               {{/if}}
             {{/let}}
           </div>
         </div>
 
+        {{!-- TODO use <PolarisImage> once implemented  --}}
```

Comment:
We might actually be able to implement this now that we have splattributes available to us 🤔